addons/website_event_track/models/event_event.py

```diff
@@ -50,6 +55,16 @@ def _compute_sponsor_count(self):
         for event in self:
             event.sponsor_count = result.get(event.id, 0)
 
+    @api.depends('event_type_id', 'website_menu')
+    def _compute_website_menu_data(self):
+        for event in self:
+            if event.event_type_id and event.event_type_id != event._origin.event_type_id:
+                event.website_track = event.event_type_id.website_track
+                event.website_track_proposal = event.event_type_id.website_track_proposal
+            elif not event.website_menu:
```

Comment:
Same remark about potential None values.

Reply:
Meh, not sure for those None / _origin. I cleaned a bit everything, will have to re-test and see if there are issues.

Reply:
I think TIC had some issues with that on the "more question types with correct answers" task.

If I remember correctly, it was quite tricky and happened randomly.
But if it works here, I guess we can ignore it.